_posts/2015-05-20-plugin-settings.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Plugin Settings
+status: public
+type: post
+published: true
+author: Go Team
+---
+
+Go is continously improving its plugin infrastructure. Last year the Plugin API change from JAVA interface based approach to JSON message based one to support multiple versions of same end-point for backward compatibility is working out quiet well.
+
```

Comment:
Knock the para above. Slightly redundant. You can state "Go is continously improving its plugin infrastructure". Starting   15.2...